lib/lib_install.php

```diff
@@ -85,14 +85,21 @@ function checkDb() {
 		$db['pdo_options'] = [];
 	}
 	$db['pdo_options'][PDO::ATTR_ERRMODE] = PDO::ERRMODE_EXCEPTION;
-	$dbBase = isset($db['base']) ? $db['base'] : '';
+	$conf->db = $db;	//TODO: Remove this Minz limitation "Indirect modification of overloaded property"
```

Comment:
I'm not sure why you need to re-set the $db in conf, but I would avoid this side effect. But maybe we can't do differently due to the actual architecture?

Reply:
Exactly: `FreshRSS_Context::$system_conf->db['key'] = 'value';` does not work due to the fact that Minz introduces a barrier with `__get()`.
The error message is in the comment: *Indirect modification of overloaded property*

Reply:
Indeed for changing the name. And yes for changing the configuration, as we are not really using its advantages (setters and getters) but we pay its drawbacks (layers that make the syntax more complex and error prone).
The comment was more to explain the strange syntax

Reply:
Name changed https://github.com/FreshRSS/FreshRSS/pull/2635/commits/41179bef9f70e233baadd3b8c4bc6974d56e02e4